jmix-flowui/flowui/src/main/resources/io/jmix/flowui/app/jmxconsole/mbean-operation-result-view.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+  ~ Copyright 2022 Haulmont.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<view xmlns="http://jmix.io/schema/flowui/view"
+      title="msg://mbeanOperationResult.title">
+    <actions>
+        <action id="closeAction" type="view_close" icon="CHECK" text="msg://close"/>
+    </actions>
+    <layout width="60em">
+        <vbox id="resultVBox" expand="resultContainer">
```

Comment:
1. disable padding
2. Currently dialog has auto height, so neither expand not scroller make sense